src/qs_dispersion_utils.F

```diff
@@ -92,7 +92,9 @@ SUBROUTINE qs_dispersion_env_set(dispersion_env, xc_section)
             CALL section_vals_val_get(pp_section, "SCALING", explicit=explicit)
             IF (.NOT. explicit) THEN
                CALL section_vals_val_get(pp_section, "REFERENCE_FUNCTIONAL", explicit=exfun)
-               CPASSERT(exfun)
+               IF (.NOT. exfun) THEN
+                  CPABORT("D2 vdW REFERENCE_FUNCTIONAL expected (example REFERENCE_FUNCTIONAL PBE). Go to https://www.chemie.uni-bonn.de/grimme/de/software/dft-d3/ for a full list of supported functionals")
```

Comment:
For aborts with long messages you can use the following:
```
CALL cp_abort(__LOCATION__, "A long error message "//&
              "with lots of valuable advice.")  
```

See also https://www.cp2k.org/dev:error_handling

Reply:
OK Thnk you for the information. I would like to make the error message more explicit because it took me some time to figure out what was the error.